applications/nrf5340_audio/src/bluetooth/bt_management/advertising/bt_mgmt_adv.c

```diff
@@ -144,7 +145,7 @@ static const struct bt_le_ext_adv_cb adv_cb = {
 #endif /* CONFIG_BT_PRIVACY */
 };
 
-static int direct_adv_create(bt_addr_le_t addr)
+static int direct_adv_create(uint8_t ext_adv_index, bt_addr_le_t addr)
```

Comment:
Would we need to bring this feature to dir-adv creation?

Reply:
If we want to support multiple CIGs we will have to include here as well. Also the API for adv is the same for BIS and CIS, so since we are changing it for BIS we need to apply it to CIS. Otherwise it would have to be hard coded to ext_adv[0] everywhere.